packer/ansible/requirements.yml

```diff
@@ -38,8 +38,6 @@
   name: disable_numa
 - src: https://github.com/GekoCloud/ansible-role-disable-thp
   name: disable_thp
-- src: https://github.com/cisagov/ansible-role-docker
-  name: docker
```

Comment:
Is this Ansible role able to be removed because it is pulled in automatically where needed?

Reply:
Yes. It was removed in https://github.com/cisagov/cyhy_amis/pull/648/commits/cd19a034f28fa0ff965056594982774c4b481e61 because it is a dependency of [cisagov/ansible-role-ncats-webui](https://github.com/cisagov/ansible-role-ncats-webui/blob/develop/meta/main.yml). I assume based on its absence as an explicit role that it is also an appropriate dependency anywhere else Docker is desired.